src/main/java/roomescape/theme/service/ThemeService.java

```diff
@@ -32,8 +36,8 @@ public List<ThemeResponse> findAll() {
     }
 
     public List<PopularThemeResponse> findAllPopular() {
-        final LocalDate startDate = java.time.LocalDate.now().minusDays(7);
-        final LocalDate endDate = LocalDate.now().minusDays(1);
+        final LocalDate startDate = java.time.LocalDate.now(clock).minusDays(7);
+        final LocalDate endDate = LocalDate.now(clock).minusDays(1);
```

Comment:
now() 를 각각 호출하기 때문에 00시 전후에 정말 운이 안좋으면(?) 의도하지 않은 상황이 발생할 수 있을 것 같아요
시간 관련에서는 항상 버그가 발생하는 경우가 많아서 주의해야합니다
현재는 크게 문제가 되는 케이스는 아니지만 예시를 들기 좋아서 말씀드려요~

```suggestion
        final LocalDate currentDate = LocalDate.now(clock);
        final LocalDate startDate = currentDate.minusDays(7);
        final LocalDate endDate = currentDate.minusDays(1);
```

Reply:
오 맞아요 저도 예전에 이 이슈로 문제가 생겼던 경험이 있습니다!

외부 거래소 API와 연동할 때 생긴 문제였는데 페이로드의 timestamp와 시그니처의 timestamp가 밀리초단위의 오차가 있어서 인증에 실패했었습니다.

확실히 주의해야겠네요. 감사합니다! 😄 

[refactor: 단일 메서드 내에서 LocalDate.now() 사용 로직 단일화](https://github.com/woowacourse/spring-roomescape-waiting/pull/258/commits/41dcd76931e0f578f2b8a92e7542cabb46467510)